### PR TITLE
Alert refactor: use `type` prop instead of `ErrorAlert`/`WarningAlert` etc

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
Similar to part of what Emma just refactored with Buttons. 
First step (in this PR) is getting rid of `ErrorAlert` and `WarningAlert` and whatnot, and instead passing a `type` prop into `Alert.jsx`. 

I figured I'll separate that in review from the next step (using children for Alerts), cause it's easier to review that way.

### CODE REVIEW

File: packages/blocks-docs/src/code_examples/ButtonIconExample.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react"
 
 const { Button } = require('blocks-react').Buttons;
-const { UpIcon, MoreIcon, SuccessIcon, CloseIcon, AddIcon, BackIcon, NextIcon, DownIcon } = require('blocks-react').Icons;
+const { UpIcon, MoreIcon, SuccessIcon, CloseIcon } = require('blocks-react').Icons;
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/code_examples/ButtonIconExample.js]: just noticed these were unused

File: packages/blocks-react/components/alerts/Alert.jsx [hunk 2]
```diff
@@ -2,8 +2,33 @@ const React = require('react');
 const PropTypes = require('prop-types');
```
Comment (MagdalenaHenke) [inline packages/blocks-react/components/alerts/Alert.jsx]: renamed this from AlertBanner to just Alert since it's at the front line now.

**Comment by MagdalenaHenke** [comment]
@emma-floored @Jinsoo94 - first step of Alert refactor gooood to go!

**Comment by Jinsoo94** [comment]
looks good to me!

**Comment by emma-floored** [comment]
👍

**Pull request merged**